doc/man7/migration_guide.pod

```diff
@@ -0,0 +1,1311 @@
+OSSL_ENCODER APIs to read and write files.
+See L<d2i_RSAPrivateKey(3)/Migration> for further details.
+
+=Head4 Deprecated low level key parameter getters
+
+Functions that access low level objects directly such as RSA_get0_n() are now
+deprecated. Applications should use one of L<EVP_PKEY_get_bn_param(3)>,
+L<EVP_PKEY_get_int_param(3)>, l<EVP_PKEY_get_size_t_param(3)>,
+L<EVP_PKEY_get_utf8_string_param(3)>, L<EVP_PKEY_get_octet_string_param(3)> or 
+L<EVP_PKEY_get_params(3)> to access fields from an EVP_PKEY.
+Gettable parameters are listed in L<EVP_PKEY-RSA(7)/Common RSA parameters>,
+L<EVP_PKEY-DH(7)/DH parameters>, L<EVP_PKEY-DSA(7)/DSA parameters>,
+L<EVP_PKEY-FFC(7)/FFC parameters>, L<EVP_PKEY-EC(7)/Common EC parameters> and
+L<EVP_PKEY-X25519(7)/Common X25519, X448, ED25519 and ED448 parameters>.
+Applications may also use EVP_PKEY_todata() to return all fields.
+
+=Head4 Deprecated low level key parameter setters
+
+Functions that access low level objects directly such as RSA_set0_crt_params()
+are now deprecated. Applications should use L<EVP_PKEY_fromdata(3)> to create
+new keys from user provided key data. Keys should be immutable once they are
+created so if required the user may use EVP_PKEY_todata(), OSSL_PARAM_merge(),
+and EVP_PKEY_fromdata() to create a modified key.
+See L<EVP_PKEY-DH(7)/Examples> for more information.
+See L<Deprecated low level key generation functions> for information on generating
+a key using parameters.
+
+EVP_PKEY_copy_parameters
+
+=Head Deprecated low level object creation
+
+Low level objects were created using methods such as RSA_new(), RSA_up_ref() and
+RSA_free(). Applications may use L<EVP_PKEY_new(3)>, L<EVP_PKEY_up_ref(3)>
+and L<EVP_PKEY_free(3)> instead. See also L<EVP_PKEY_CTX_new_from_name(3)> and
+L<EVP_PKEY_CTX_new_from_pkey(3)>
+
+EVP_PKEY's may be created in a variety of ways:
+See also L<Deprecated low level key generation functions>,
+L<Deprecated low level key reading and writing functions> and
+L<Deprecated low level key parameter setters>.
+
+=head4 Deprecated low level encryption functions
+
+Many low level functions has been informally discouraged from use for
+a long time. Applications should use the high level EVP APIs, e.g.
+L<EVP_EncryptInit_ex(3)>, L<EVP_EncryptUpdate(3)>, L<EVP_EncryptFinal_ex(3)>,
+OR L<EVP_DecryptInit_ex(3)>, L<EVP_DecryptUpdate(3)>, L<EVP_DecryptFinal_ex(3)>.
+
+=head4 Deprecated low level digest functions
+
+Use of these low level digest functions (such as SHA1_init()) have been
+informally discouraged for a long time.  Applications should use the
+EVP_DigestInit_ex(3), EVP_DigestUpdate(3) and EVP_DigestFinal_ex(3) functions
+instead.
+
+Many low level functions has been informally discouraged from use for
+a long time. Applications should use the high level EVP APIs, e.g.
+L<EVP_EncryptInit_ex(3)>, L<EVP_EncryptUpdate(3)>, L<EVP_EncryptFinal_ex(3)>,
+OR L<EVP_DecryptInit_ex(3)>, L<EVP_DecryptUpdate(3)>, L<EVP_DecryptFinal_ex(3)>.
```

Comment:
Seems to be repeated text

Reply:
In which sense do you see this as repeated?

Reply:
In the sense that the exact same paragraph appears 12 lines earlier.

Reply:
Removed second copy.